mpb/fields.c

```diff
@@ -706,7 +706,7 @@ number get_energy_point(vector3 p)
      return f_interp_val(p, mdata, (real *) curfield, 1, 0);
 }
 
-void get_bloch_field_point_(scalar_complex *field, vector3 p)
+void get_bloch_field_point_(scalar_complex field[static 3], vector3 p)
```

Comment:
```suggestion
void get_bloch_field_point_(scalar_complex field[3], vector3 p)
```

This `static` is a C99 feature, but we don't currently require C99 (which involves setting flags on some compilers, and I don't know if it's supported by all C++ compilers.

Reply:
Okay, makes sense (I had tried to look up which C versions MPB were supporting but could not immediately find out - so I just assumed C99+; though now I think I remember you telling me a similar thing before).